maintainers/maintainer-list.nix

```diff
@@ -16266,6 +16266,12 @@
     github = "sweenu";
     githubId = 7051978;
   };
+  swesterfeld = {
```

Comment:
Split this in a separate commit - no need to open a PR.

Reply:
Ok, I made this a separate commit and merged all your suggestions into one commit and force pushed the result.